nss-day-cohort-60/rare-python-server-tyrannical-tators
# updated do_GET to work with new fetch URL in subscribed posts

Closes #28 
# Description of Proposed Changes

Update to do_GET to work with new fetch url in getSubscriptions()

# Steps to Test

1. fetch[ jes-filter-categories ](https://github.com/nss-day-cohort-60/rare-python-server-tyrannical-tators/compare/main...jes-filter-categories?expand=1)branch and switch to it
2. run your debugger, make sure you have subscriptions in your SQL DB, if not run this INSERT statement in loaddata.sql
INSERT INTO Subscriptions ('follower_id', 'author_id', 'created_on') Values (1, 2, 2023-01-30);

3. Open Postman and GET http://localhost:8088/posts?follower_id=1
4. verify you get back only posts with a user id (author) that matches who your user (follower) follows (it's confusing, ask me questions if you need to)
